extensions/amp-story/1.0/amp-story-reaction.css

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
Empty as of now since not sure what styles will be shared among reaction components. Could delete if not useful.